packaging/google_colab_install.md

```diff
@@ -16,200 +14,39 @@ This page provides instructions to build, install and use the AIMET software in
-
-!dpkg -i cuda-repo-ubuntu1804_10.0.130-1_amd64.deb
-
-!apt-get update
-
-!wget http://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64/nvidia-machine-learning-repo-ubuntu1804_1.0.0-1_amd64.deb
-
-!apt install ./nvidia-machine-learning-repo-ubuntu1804_1.0.0-1_amd64.deb
-
-!apt-get update
-
-!apt install cuda-cublas-10-0 cuda-cufft-10-0 cuda-curand-10-0 cuda-cusolver-10-0
-
-!apt-get update && apt install cuda-cusparse-10-0 libcudnn7=7.6.2.24-1+cuda10.0 libnccl2=2.4.8-1+cuda10.0  cuda-command-line-tools-10.0
-
-!pip3 install scipy==1.1.0
-
-!pip3 install protobuf==3.7.1
-
-!pip3 install scikit-learn==0.19.1
-
-!pip3 install tb-nightly==1.14.0a20190517
-
-!pip3 install tensorboardX==1.7
-
-!pip3 install https://download.pytorch.org/whl/cu100/torch-1.4.0%2Bcu100-cp36-cp36m-linux_x86_64.whl
-
-!pip3 install https://download.pytorch.org/whl/cu100/torchvision-0.5.0%2Bcu100-cp36-cp36m-linux_x86_64.whl
-
-!pip3 install --upgrade pip
-
-!pip3 install tensorflow-gpu==1.15.0
-
-!pip3 install future==0.17.1
-
-!pip3 install tensorboard==1.14
 
-!pip3 install bokeh==1.2.0
-
-!pip3 install pandas==0.22.0
-
-!pip3 install holoviews==1.12.7
-
-!pip3 install --no-deps bokeh==1.2.0 hvplot==0.4.0
-
-!pip3 install jsonschema==3.1.1
-
-!pip3 install osqp onnx
-
-!ln -s /usr/local/cuda-10.0 /usr/local/cuda
-
-!apt-get update && apt-get install -y libjpeg8-dev
-
-!ln -s /usr/lib/x86_64-linux-gnu/libjpeg.so /usr/lib
-
-!apt install zlib1g-dev
-
-!pip3 uninstall Pillow && pip3 install Pillow-SIMD==6.0.0.post0
-
-!pip3 uninstall pytest
-
-!pip3 install pytest
-
-!pip3 install setuptools==41.0.1
-
-!pip3 install keras==2.2.4
-
-%rm -rf /usr/local/bin/python
-
-!ln -s /usr/bin/python3 /usr/local/bin/python
-```
-## Reset Google colab environment
-Please restart Google runtime environment from below menu option:
+Please **restart** Google runtime environment when prompted or from below menu option:
 
 Runtime -> Restart runtime
 
-## AIMET build and installation
-Please run below commands to fetch AIMET, and googletest from github repo, and compile, and install AIMET.
-
-```
-%cd /content/
-
-!mkdir aimet_code
-
-%cd aimet_code
-
-!git clone https://github.com/quic/aimet.git
 
-%cd aimet
-
-%mkdir -p ./ThirdParty/googletest
-
-%pushd ./ThirdParty/googletest
-
-!git clone https://github.com/google/googletest.git -b release-1.8.0 googletest-release-1.8.0
-
-%popd
-
-%cd /content/aimet_code
-
-%mkdir build
-
-%cd build
-
-!cmake -DCMAKE_EXPORT_COMPILE_COMMANDS=ON ../aimet
-
-!make -j 8
-
-!make install
-```
-
-
-## Configure LD_LIBRARY_PATH and PYTHONPATH
+## Configure
 
 ```python
 import sys
-
-sys.path.append(r'/content/aimet_code/build/staging/universal/lib/python')
-
-sys.path.append(r'/content/aimet_code/build/staging/universal/lib/x86_64-linux-gnu')
-
-sys.path.append(r'/usr/local/lib/python3.6/dist-packages')
-
-sys.path.append(r'/content/aimet_code/build/artifacts')
+sys.path.append('/usr/local/lib/python3.6/dist-packages/aimet_common')
```

Comment:
Why are we doing this? /usr/local/lib/python3.6/dist-packages/ should automatically be in the search path. We don't need to add it to sys,path.

Reply:
ok, it was in the original instruction, will test and remove it